htdocs/core/lib/functions.lib.php

```diff
@@ -8633,6 +8633,9 @@ function getCommonSubstitutionArray($outputlangs, $onlykey = 0, $exclude = null,
 				}
 				if (is_object($object) && $object->element == 'facture') {
 					$substitutionarray['__URL_INVOICE__'] = DOL_MAIN_URL_ROOT."/compta/facture/card.php?id=".$object->id;
+					$substitutionarray['__FACDATE__'] = isset($object->date) ? dol_print_date($object->date, 'daytext', 0, $outputlangs) : '';
```

Comment:
We should already have this. See
https://wiki.dolibarr.org/index.php?title=Variable_substitution_system#Objects_with_amounts